packages/vanilla/src/components/card/bannerCard.stories.js

```diff
@@ -0,0 +1,47 @@
+export default {
+  title: 'Patterns/Card',
+  parameters: {
+    layout: 'centered'
+  },
+  argTypes: {
+    title: {
+      name: 'Title',
+      description: 'Set the title in the banner area of the card',
+      control: 'text'
+    },
+    modifier: {
```

Comment:
Can we call this "Variant" instead of "Modifier"?

Reply:
Changed to 'variant'